inference-engine/ie_bridges/python/install/setup.py.in

```diff
@@ -0,0 +1,287 @@
+
+
+class CustomBuild(build):
+    """Custom implementation of build_clib"""
+    def run(self):
+        self.run_command('build_clib')
+        build.run(self)
+
+
+class CustomInstall(install):
+    """Enable build_clib during the installation"""
+    def run(self):
+        self.run_command('build_clib')
+        install.run(self)
+
+
+class PrepareLibs(build_clib):
+    """Collect prebuilt libraries. Install them to the temp directories, set rpath."""
+    def run(self):
+        for comp, comp_data in CMAKE_INSTALL_CFG.items():
+            self.announce(f"Installing {comp}", level=3)
+            self.spawn(["cmake",
+                        "--install", CMAKE_BUILD_DIR,
+                        "--prefix", comp_data.get("prefix"),
+                        "--component", comp_data.get("name")])
+
+        self.announce("Configuring libraries", level=3)
+        if sys.platform != "win32":
+            rpath = ""
+            lib_pattern = ""
+            if sys.platform == "linux":
+                rpath = "$ORIGIN"
+                lib_pattern = "*.so"
+            elif sys.platform == "darwin":
+                rpath = "@loader_path"
+                lib_pattern = "*.dylib"
+            for lib in RELOCATABLE_LIBS:
+                for path in Path(lib).glob(lib_pattern):
+                    set_rpath(rpath, path)
+
+
+class CopyExt(build_ext):
+    """Copy extension files to the build directory"""
+    def run(self):
+        for extension in self.extensions:
+            if not isinstance(extension, PrebuiltExtension):
+                raise DistutilsSetupError(
+                    f"copy_ext can accept PrebuiltExtension only, but got {extension.name}")
+            src = extension.sources[0]
+            dst = self.get_ext_fullpath(extension.name)
+            os.makedirs(os.path.dirname(dst), exist_ok=True)
+            #setting relative path to find dlls
+            if sys.platform != "win32":
+                rpath = os.path.relpath(BINDINGS_DIR, os.path.dirname(src))
+                if sys.platform == "linux":
+                    rpath = os.path.join("$ORIGIN", rpath, WHEEL_LIBS_INSTALL_DIR)
+                elif sys.platform == "darwin":
+                    rpath = os.path.join("@loader_path", rpath, WHEEL_LIBS_INSTALL_DIR)
+                set_rpath(rpath, src)
+
+            copy_file(src, dst, verbose=self.verbose, dry_run=self.dry_run)
+
+
+def is_tool(name):
+    """Check if the command-line tool is available"""
+    try:
+        devnull = subprocess.DEVNULL
+        subprocess.Popen([name], stdout=devnull, stderr=devnull).communicate()
+    except OSError as error:
+        if error.errno == errno.ENOENT:
+            return False
+    return True
+
+
+def set_rpath(rpath, executable):
+    """Setting rpath for linux and macOS libraries"""
+    print(f"Setting rpath {rpath} for {executable}")
+    cmd = ""
+    rpath_tool = ""
+    if sys.platform == "linux":
+        rpath_tool = "patchelf"
```

Comment:
On Ubuntu 18.04: 
_Configuring libraries
Setting rpath $ORIGIN for libs.core/deployment_tools/inference_engine/lib/intel64/libinference_engine_onnx_reader.so
Could not found patchelf on the system, please make sure that this tool is installed
inference-engine/ie_bridges/python/install/CMakeFiles/ie_wheel.dir/build.make:80: recipe for target 'ie_wheel' failed
make[2]: *** [ie_wheel] Error 1_

Do we need to install additional  prerequisites to build a wheel ? If so, should add it into `install_build_dependencies.sh`?

Reply:
These are distribution-specific tools and it is better just to mention these requirements in the docs

Reply:
@mryzhov ,
This is not best option from usability point of view. Open source developers are required to run one more step to install  patchelf before building a wheel package. Is it already documented anywhere? 

@openvinotoolkit/openvino-admins ,
What is BKM for this? Please advise.

Reply:
Added to python API readme